Taskfile.yml

```diff
@@ -263,3 +275,48 @@ tasks:
     dir: docker
     cmds:
       - docker compose -f docker-compose.yml -p mealie up -d --build
+
+  e2e:build-image:
+    desc: builds the e2e test docker image
+    deps:
+       - py:package
+    cmds:
+      - docker build --tag mealie:e2e --file docker/Dockerfile --build-context packages=dist .
+    sources:
+      - docker/Dockerfile
+      - dist/**
+
+  e2e:start-server:
+    desc: Builds the image and starts the containers for e2e testing
+    dir: tests/e2e/docker
+    deps:
+       - e2e:build-image
+    vars:
+      WAIT_UNTIL_HEALTHY: '{{if .WAIT_UNTIL_HEALTHY}}--wait{{else}}{{end}}'
+    cmds:
+      - docker compose up -d {{.WAIT_UNTIL_HEALTHY}}
+
+  e2e:stop-server:
+    desc: Shuts down the e2e testing containers
+    dir: tests/e2e/docker
+    cmds:
+      - docker compose down --volumes
+
+  e2e:test:
+    desc: runs the e2e tests
+    dir: tests/e2e
+    deps:
+      - setup:e2e
+    vars:
+      PREVENT_REPORT_OPEN: '{{if .PREVENT_REPORT_OPEN}}PLAYWRIGHT_HTML_OPEN=never{{else}}{{end}}'
+    cmds:
+      - '{{.PREVENT_REPORT_OPEN}} yarn playwright test'
```

Comment:
comment: playwright defaults to serving the report (staying active in the terminal) if any tests fail. You then need to stop it with Ctrl+C. 
This conflicts with `task`'s handling of the interrupt signal, so when you run the all-in-one script `e2e`, and then you stop the playwright report server with Ctrl+C, `task` thinks you're aborting the tasks and shows an error message.

Using `PREVENT_REPORT_OPEN`, we can tell playwright not to serve the report (you can still open last report afterwards with `yarn playwright show-report`)